smoke_test

```diff
@@ -124,9 +125,13 @@ private
   end
 
   def request_token
-    visit '/'
```

Comment:
To clarify a bit, it seems to me that `visit '/'` should be at the top, before the page content check, and the `visit '/'` at line 130 is redundant.